### DISCUSSION TITLE
高刷显示器播放视频卡顿

### DISCUSSION THREAD

**Discussion by RoachLin** [diana7127/mpv.net-DW#31] [Q&A]
我的显示器是4K144的，我用的软件是2.0.0版本。
下载解压后我直接播放一个4K视频，发现卡顿，后来在设置-视频-第一个选项那里，把重采样改为了第一个默认，才恢复正常。
所以我想问一下，有没有办法在开启重采样的情况下，使得高刷显示器播放视频不卡顿？

**Comment by moleoxa** [comment]
那是因为你显卡性能不够。

**Reply by RoachLin** [reply]
那不可能啊，我显卡是3060tig6x，而且我看任务管理器才十几二十几的显卡占用，根本没跑满

**Comment by moleoxa** [comment]
反正我不开重采样是因为开了的情况下只要倍速显卡占用就蹭蹭涨。
